nixos/modules/services/web-apps/mealie.nix

```diff
@@ -39,6 +39,15 @@ in
       };
     };
 
+    extraOptions = lib.mkOption {
+      type = lib.types.listOf lib.types.str;
+      default = [ ];
+      example = [ "--forwarded-allow-ips=10.44.0.11" ];
```

Comment:
I don't think it's a good example, as the `--forwarded-allow-ips` argument is what is set by the `trustedProxies` option here

Reply:
Done!